api/python/provisioner/commands/check.py

```diff
@@ -1104,6 +1051,59 @@ def _controller_ip(*, args: str) -> Union[CheckEntry, List[CheckEntry]]:
 
         return res
 
+    @staticmethod
+    def _validate_bmc_check(check: str):
+
+        if check == "stonith":
```

Comment:
Use constants here for `stonith` and `accessible`. Also I would recommend you to use `if-elif-else` here, and raise an exception if the user passed something different from these two variants:
```python
if check == "stonith":
     bmc_check = cfg.Checks.BMC_STONITH.value
elif check == "accessible":
     bmc_check = cfg.Checks.BMC_ACCESSIBILITY.value
else:
    raise Exception("Unsupported check value")
```

Reply:
comment is addressed